src/filters/base_filter.class.js

```diff
@@ -20,6 +20,8 @@ fabric.Image.filters.BaseFilter = fabric.util.createClass(/** @lends fabric.Imag
    */
   type: 'BaseFilter',
 
+  webGlPrecision: 'highp',
```

Comment:
can we make this fabric global property? as we have fabric.webGlprecision. texture size is also there

Reply:
on it